frontend/packages/process-editor/src/utils/bpmn/BpmnActionModeler.ts

```diff
@@ -0,0 +1,109 @@
+import type { ModdleElement } from 'bpmn-js/lib/BaseModeler';
```

Comment:
Lots of CodeCov comments in this file, is it possible to test, or avoided because of the package we use? 
If so, should we add a comment explaining it for future developers to see? 😄

Reply:
Added a comment at the top of the file. 😊